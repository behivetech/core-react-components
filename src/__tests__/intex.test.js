import index from '../index';
// jest.mock('../index');

describe('index', () => {
	it('should contain all of the components', () => {
		expect(Object.keys(index)).toMatchSnapshot();
	});
});