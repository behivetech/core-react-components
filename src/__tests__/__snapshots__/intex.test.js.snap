// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should contain all of the components 1`] = `
Array [
  "Button",
  "Checkbox",
  "ButtonSubmitControlled",
  "CheckboxControlled",
  "Form",
  "FormControlled",
  "Select",
  "SelectControlled",
  "Switch",
  "SwitchControlled",
  "TextField",
  "TextFieldControlled",
]
`;
